examples/chip-tool/main.cpp

```diff
@@ -148,10 +154,11 @@ static void HandleDataModelMessage(System::PacketBuffer * buffer)
     System::PacketBuffer::Free(buffer);
 }
 
-static void OnMessage(DeviceController::ChipDeviceController * deviceController, void * appReqState, System::PacketBuffer * buffer)
+void OnMessage(DeviceController::ChipDeviceController * deviceController, void * appReqState, System::PacketBuffer * buffer)
 {
     size_t data_len    = buffer->DataLength();
     waitingForResponse = false;
```

Comment:
The thread must hold cvWaitingForResponseMutex for this store (and can remove the std::atomic since it won't be helpful here).

Reply:
Done